src/main/java/com/ardaslegends/domain/war/War.java

```diff
@@ -44,8 +44,13 @@ public class War extends AbstractDomainObject {
     @NotNull
     private OffsetDateTime startDate;
 
+    @Setter
     private OffsetDateTime endDate;
```

Comment:
Setter should be private, so that wars can only be ended through the end() method

Reply:
Done